routes/listingRoute.js

```diff
@@ -27,33 +27,44 @@ listingRoute.get("/new", (req, res) => {
 listingRoute.get("/:id", wrapAsync(async (req, res) => {
   let { id } = req.params;
   const listing = await Listing.findById(id);
+  if(!listing) {
+    req.flash("error", "Listing doesn't exist anymore :(");
+    res.redirect("/listings");
```

Comment:
You are flashing and then redirecting.. will the user be able to see this? Can you add a screenshot demonstrating this flow